.github/workflows/checking.yml

```diff
@@ -11,7 +11,6 @@ jobs:
       matrix:
         toolchain:
           - 1.42.0
```

Comment:
can we at least check that the project compiles on the latest nightly or does that break as well?

Reply:
I can try